pro/src/components/OfferIndividualForm/utils/setFormReadOnlyFields.ts

```diff
@@ -42,16 +42,7 @@ const setFormReadOnlyFields = (
   } else {
     readOnlyField = [
       ...readOnlyField,
-      ...[
-        'categoryId',
-        'subcategoryId',
-        'offererId',
-        'venueId',
-        'showType',
-        'showSubType',
-        'musicType',
-        'musicSubType',
-      ],
```

Comment:
Pareil que le commentaire précédents modifs commit par erreur ?

Reply:
ça a été ajouté par @mageoffray , a voir avec lui

Reply:
@mageoffray est OFF, je pense qu'il a commit ces modifications par erreur sur cette branche. Je propose de reset ces modifications si la PR doit passer pour la MES de cette ité, sinon on attend son retour